completion/git-machete.completion.fish

```diff
@@ -1,189 +1,205 @@
-# This file will be loaded by git.fish completions and also provide all __fish_git_* functions here.
```

Comment:
Restore `git-machete.fish`, identical to `git-machete.completion.fish` &mdash; for the sake of compatibility in Homebrew formula (and maybe other packages? TBC)

Reply:
#1010